src/node_crypto.cc

```diff
@@ -3952,6 +3953,41 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment *env, v8::Local<v8::Object> options,
+                            int *padding, int *saltlen) {
+  Local<Value> key = String::NewFromUtf8(env->isolate(), "padding");
+  MaybeLocal<Value> maybePadding = options->Get(key);
+  if (!maybePadding.IsEmpty()) {
```

Comment:
> Considering that I `return 0` after throwing an exception (so `0 <=> error`), I should probably `return 1` here as it should be optional to specify the padding.

In the case where `maybePadding` is empty you can safely assume that an exception is pending (e.g. if the getter for `"padding"` threw an error – which is obviously mostly hypothetical, but since it’s an object passed in from userland, it’s possible). So, returning `0` and returning back to JS as quickly as possible should be the right thing to do.

If the property is missing, you’ll get an `Undefined` value, not an empty `MaybeLocal`.